download.html

```diff
@@ -125,18 +125,18 @@ <h3 class="text-dark">Schedule</h3>
                       <td>7</td>
                       <td>8 / 9 / 10 / 11</td>
                       <td class="text-success">Preview</td>
-                      <td></td>
                       <td>4.3.0.RC1</td>
-                      <td>Jan '20</td>
+                      <td>4.3.0.RC2</td>
+                      <td>Jul '20</td>
                     </tr>
                     <tr>
                       <td>4.2.x</td>
                       <td>6</td>
                       <td>8 / 9 / 10 / 11</td>
                       <td class="text-success">Stable</td>
-                      <td>4.2.8</td>
                       <td>4.2.9</td>
-                      <td>Mar '20</td>
+                      <td>4.2.10</td>
+                      <td>Oct '20</td>
```

Comment:
I would say more about August for 4.2.10. Thoughts ?

Reply:
Agree, update done.